.github/workflows/deploy.yml

```diff
@@ -18,40 +18,68 @@ jobs:
         id: set-matrix
         run: |
           MATRIX_JSON=$(jq -c . < deployments.json)
-          echo "::set-output name=matrix::$MATRIX_JSON"
+          run: echo "matrix=$MATRIX_JSON" >> $GITHUB_OUTPUT
 
   deploy:
     needs: prepare-matrix
     runs-on: ubuntu-latest
+    
+    concurrency:
+      group: deploy-${{ matrix.folder }}
+      # Wait for prior deployments to finish, otherwise new deployments will be rejected
+      cancel-in-progress: false
```

Comment:
```suggestion
    # concurrency:
    #   group: deploy-${{ matrix.folder }}
    #   # Wait for prior deployments to finish, otherwise new deployments will be rejected
    #   cancel-in-progress: false
```